Makefile

```diff
@@ -396,8 +403,8 @@ VERSION = $(shell cat ecs-init/ECSVERSION)
 	mkdir -p BUILDROOT
 	./scripts/update-version.sh
 	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init scripts README.md
-	cp -r packaging/generic-deb-integrated/debian ecs-init scripts misc agent agent-container amazon-ecs-cni-plugins amazon-vpc-cni-plugins README.md VERSION BUILDROOT
-	cd BUILDROOT && debuild -uc -us --lintian-opts --suppress-tags bad-distribution-in-changes-file,file-in-unusual-dir
+	cp -r packaging/generic-deb-integrated/debian Makefile ecs-init scripts misc agent agent-container amazon-ecs-cni-plugins amazon-vpc-cni-plugins README.md VERSION GO_VERSION BUILDROOT
+	cd BUILDROOT && dpkg-buildpackage -uc -us -b
```

Comment:
Since we're using `dpkg-buildpackage` and not `debuild`, does the built in `debsign` stuff still happen?

Reply:
removed the -us (unsigned-source) to force dpkg-buildpackage to do the signing stuff
https://man7.org/linux/man-pages/man1/dpkg-buildpackage.1.html